doc/howtos/building-images.rst

```diff
@@ -257,6 +285,68 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
 
+Adding a Custom Layer in Ostro OS
+---------------------------------
+
+.. _`Creating Your Own Layer`: http://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#creating-your-own-layer
+.. _`Open Embedded Layers Index`:  http://layers.openembedded.org/layerindex/branch/master/layers/
+
+The Yocto Project documentation explains the steps you'd follow for `Creating Your Own Layer`_. 
+
+#. Within your
+   cloned copy of ``ostro-os``, here's how you can easily add a custom layer into your Ostro OS build:::
+
+      $ git clone <meta-custom-layer-name>     # clone the git repo for your custom layer 
+      $ source oe-init-build-env               # initialize the build environment 
+
+#. Add a line to your ``build/conf/bblayers.conf`` file adding the layer:::
+
+      BBLAYERS += "/PATH/TO/LAYERS/meta-custom-layer-name"
+
+   Alternatively, you can use the ``bitbake-layers`` command to manipulate the ``bblayers.conf`` file for you:::
+
+     $ bitbake-layers add-layer meta-custom-layer-name 
+     $ bitbake-layers show-layers                        # verify bitbake sees the layer
+
+#. If this new layer depends on others that aren't already included in the build, you'll 
+   need to add additional ``BBLAYERS += "..."`` lines (either manually or by using
+   the ``bitbake-layers add-layer`` command)
+
+#. Add this to the end of your ``build/conf/local.conf`` file:::
```

Comment:
I'd prefer we drop the build/ part here and make it ...your ``conf/local.conf``...

build/ is the value by default but can be tweaked by folks, besides you're placed in that folder by default already so no need to mention the path